app/code/Magento/ReviewGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,78 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+interface ProductInterface {
+    rating_summary: Float! @doc(description: "The average of all the ratings given to the product.") @resolver(class: "Magento\\ReviewGraphQl\\Model\\Resolver\\Product\\RatingSummary")
+    review_count: Int! @doc(description: "The total count of all the reviews given to the product.") @resolver(class: "Magento\\ReviewGraphQl\\Model\\Resolver\\Product\\ReviewCount")
+    reviews(
+        pageSize: Int = 20 @doc(description: "Specifies the maximum number of results to return at once."),
+        currentPage: Int = 1 @doc(description: "Specifies which page of results to return."),
+    ): ProductReviews! @doc(description: "The list of products reviews.") @resolver(class: "Magento\\ReviewGraphQl\\Model\\Resolver\\Product\\Reviews")
+}
+
+type ProductReviews {
+    items: [ProductReview]! @doc(description: "An array of product reviews.")
+    page_info: SearchResultPageInfo! @doc(description: "Metadata for pagination rendering.")
+}
+
+type ProductReview @doc(description: "Details of a product review") {
+    product: ProductInterface! @doc(description: "Contains details about the reviewed product") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product")
+    summary: String! @doc(description: "The summary (title) of the review")
+    text: String! @doc(description: "The review text.")
+    nickname: String! @doc(description: "The customer's nickname. Defaults to the customer name, if logged in")
+    created_at: String! @doc(description: "Date indicating when the review was created.")
+    average_rating: Float! @doc(description: "The average rating for product review.") @resolver(class: "Magento\\ReviewGraphQl\\Model\\Resolver\\Product\\Review\\AverageRating")
```

Comment:
average_rating can be null when ratings are not visible on storefront. querying for average rating might throw an error. cc: @paliarush

Reply:
Solution is to make average_rating as optional.

Reply:
I think that we may not return any product review if they are disabled and 0 for `rating_summary` and `review_count`. Is a such behavior correct @nrkapoor?
Or we should always return product reviews, even the reviews are disabled?

Reply:
@eduard13 @prabhuram93 if reviews are disabled then we should not return it via GraphQL.

Reply:
The implementation has been slightly improved, by checking whether the reviews are enabled or not. So, currently we are returning 0 for `rating_summary` and `review_count` as they aren't optional and we're required to return a numeric value.
As for `average_rating`, we should be fine here, as we won't return any item and it shouldn't occur any problems.

Reply:
average_rating can be null when a review for a product is added without a rating. Try disabling rating, add a review with just nickname, summary and text. when we query it back average_rating will be null. I checked this case and is currently throwing an internal server error.

Reply:
Thank you for this test case, have added the fix for it as well.